src/system/ThemeModule/Controller/VarController.php

```diff
@@ -0,0 +1,84 @@
+<?php
+/**
+ * Copyright Zikula Foundation 2015 - Zikula Application Framework
+ *
+ * This work is contributed to the Zikula Foundation under one or more
+ * Contributor Agreements and licensed to You under the following license:
+ *
+ * @license GNU/LGPLv3 (or at your option, any later version).
+ *
+ * Please see the NOTICE file distributed with this source code for further
+ * information regarding copyright and licensing.
+ */
+
+namespace Zikula\ThemeModule\Controller;
+
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route; // used in annotations - do not remove
+use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\HttpFoundation\Request;
+use Zikula\Core\Controller\AbstractController;
+use Zikula\Core\Theme\Annotation\Theme; // used in annotations - do not remove
+
+class VarController extends AbstractController
+{
+    /**
+     * Configure a theme's variables based on provided .yml definitions for each field.
+     * @Route("/admin/var/{themeName}")
```

Comment:
Nevermind, if I read the next line before commenting, I would know better :boat: